scripts/orca-singularity

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+if [ "$1" = "-c" ];then
+        cmd=`echo $2 | grep "rsync\|scp"`
+        if [ -z "$cmd" ]; then
+                echo "======================================================="
+                echo "           Welcome to BCGSC ORCA Service          "
+                echo "Only direct rsync or scp file transfer commands allowed"
+                echo "======================================================="
+        else
+                exec $2
+        fi
+else
+        USER=$(id -un)
+        LOG_FILE=/orca_repo/access_log/singularity_login.log
+
+        echo "Welcome to GSC ORCA Phase 1, USER $(id -un), Your home directory is mounted under /home/$USER"
+        echo "Currently installed tools can be found via 'brew list' and 'ls ~/Software/bin' in shell"
```

Comment:
What's this `~/Software/` directory, and is it documented? I don't see it in the `bcgsc/orca` Docker image.